src/lib/parse/forbiddenChars.cpp

```diff
@@ -44,7 +44,7 @@ inline static bool commonForbidden(char c)
   case '>':
   case '"':
   case '\'':
-  case '=':
+//  case '=':
```

Comment:
This breaks the forbidden characters functionality (see https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html). Unfortunatelly, that means that this PR cannot be merged in its current form into the official Orion master (although of course the branch of this PR is usable for special addapations of Orion you may need, as I understand by your question at http://stackoverflow.com/questions/44049667/orion-notifications-with-basic-auth)